File: Open3D/11_reconstruccion_superficies/python/main.py
import open3d as o3d
pc=o3d.io.read_point_cloud("input.ply"); pc.estimate_normals(); mesh,density=o3d.geometry.TriangleMesh.create_from_point_cloud_poisson(pc,depth=9); mesh.remove_degenerate_triangles(); o3d.io.write_triangle_mesh("poisson.ply",mesh)

File: Open3D/11_reconstruccion_superficies/cpp/main.cpp
#include <open3d/Open3D.h>
int main(){ auto pc=open3d::io::CreatePointCloudFromFile("input.ply"); if(!pc) return 1; pc->EstimateNormals(); auto result=open3d::geometry::TriangleMesh::CreateFromPointCloudPoisson(*pc,9); auto mesh=std::get<0>(result); mesh->RemoveDegenerateTriangles(); open3d::io::WriteTriangleMesh("poisson.ply",*mesh); }
